### ISSUE TITLE
Page Following (notification of changes)

### ISSUE THREAD

**Issue by swunderlee** [massgov/feature-requests#265]
I've been asked if there is a way for the public to be notified of additions/changes to mass.gov. The question was specific to public meetings that are posted as Events. MassDOT holds meetings for projects and we have been asked if there was a way to track when a new event is added to the site. Since we don't have a calendar on the site, I though following the page might be a good first step.

**Comment by cmrodriguez26** [comment]
I agree, notifications to the public for events posted would be extremely helpful.

**Comment by cmrodriguez26** [comment]
This would be extremely helpful for our Boards and Commissions interested parties.
